mmtbx/ions/identify.py

```diff
@@ -159,6 +158,11 @@ def ion_master_phil():
 WATER, WATER_POOR, LIGHT_ION, HEAVY_ION = range(4)
 NUC_PHOSPHATE_BINDING = ["MG", "CA", "MN"]
 
+def _sign(x):
+  if x > 0: return 1
+  if x < 0: return -1
+  return 0
+
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.